pkg/tfgen/docs_test.go

```diff
@@ -624,6 +624,38 @@ func TestArgumentRegex(t *testing.T) {
 	}
 }
 
+func TestArgumentRegexAuto(t *testing.T) {
+	tests := []struct {
+		name     string
+		input    []string
+		expected autogold.Value
+	}{
+		{
+			name: "newline after bullet",
+			input: []string{
+				"",
+				"The following arguments are supported:", "",
+				"* `versioning` - (Optional) A state of [versioning](https://www.scaleway.com/en/docs/storage/object/how-to/use-bucket-versioning/). The `versioning` object supports the following:", "",
+				"    * `enabled` - (Optional) Enable versioning. Once you version-enable a bucket, it can never return to an unversioned state. You can, however, suspend versioning on that bucket.", "",
```

Comment:
```suggestion
				"The following arguments are supported:", 
				"",
				"* `versioning` - (Optional) A state of [versioning](https://www.scaleway.com/en/docs/storage/object/how-to/use-bucket-versioning/). The `versioning` object supports the following:", 
				"",
				"    * `enabled` - (Optional) Enable versioning. Once you version-enable a bucket, it can never return to an unversioned state. You can, however, suspend versioning on that bucket.", 
				"",
```

Hiding the newlines makes this really hard to read.